pages/courses/[courseSlug]/material/lectures/[lecturesSlug].tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import CourseArticle from "../../../../Components/CourseAssigments/CourseArticle";
 import SideMenu from "../../../../Components/Courses/SideMenu";
-import BreadCrumb from "../../../../Components/Courses/BreadCrumb";
+import BreadCrumb from "../../../../Components/BreadCrumb/BreadCrumb";
```

Comment:
Alias the "src" folder to "@" in tsconfig.json and vite.config.ts to avoid having to use deep relative imports "../../../"